client/src/store/account.module.js

```diff
@@ -65,11 +80,20 @@ const mutations = {
   },
   logoutSuccessful(state) {
     state.user = {...defaultState.user};
+    state.department = {...defaultState.department};
   },
   logoutFailure() {
     //TODO: Handle error
     // console.log('ERROR: ', e);
   },
+  departmentFetchSuccessful(state, department) {
+    state.department = {...defaultState.department, ...department};
+    state.department.loaded = true;
+  },
+  departmentFetchFailure() {
+    //TODO: Handle error
+    // console.log('ERROR: ', e);
```

Comment:
?

Reply:
This is an empty mutation (reducer for the React familiar) that will handle the catch block in the future